proto/tendermint/types/types.proto

```diff
@@ -141,6 +142,11 @@ message SignedHeader {
   Commit commit = 2;
 }
 
+message LightBlock {
+  SignedHeader                  signed_header = 1;
+  tendermint.types.ValidatorSet validator_set = 2;  
```

Comment:
```suggestion
  tendermint.types.ValidatorSet validator_set = 2 [(gogoproto.nullable) = false];  
```

Reply:
I get a bunch of errors when I add this. 
```
tendermint/types/types.proto:146:52: Expected type name.
tendermint/types/types.proto:147:52: Expected type name.
tendermint/abci/types.proto:9:1: Import "tendermint/types/types.proto" was not found or had errors.
tendermint/abci/types.proto:79:3: "tendermint.types.Header" is not defined.
```

Reply:
Could we do a nil check instead? nullable will become obsolete with optional field once implemented across languages.

Reply:
We already have nil checks when going from proto to light block